kubernetes-aws-typescript/Pulumi.yaml

```diff
@@ -0,0 +1,10 @@
+name: ${PROJECT}
+description: ${DESCRIPTION}
+runtime: nodejs
+template:
+  description: A Nodejs program to create a Kubernetes cluster on AWS
```

Comment:
Would this be instead `A TypeScript program...`?

Reply:
Fixed

Reply:
Fixed in 73aaab4